src/main/java/bio/terra/cli/command/user/sshkey/Generate.java

```diff
@@ -30,8 +30,7 @@ protected void execute() {
     confirmationPrompt.confirmOrThrow(
         "Generating a new Terra SSH key will replace the old Terra SSH key if it exists. "
             + "You must associate the new SSH public key with your GitHub account using "
-            + "https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent. "
-            + "Are you sure you want to proceed (y/N)?",
+            + "https://github.com/settings/keys. Are you sure you want to proceed (y/N)?",
```

Comment:
Why did this change? I followed https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent when I set up my key

For me, it was helpful that cli printed https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent

Reply:
i thought it's more straightforward to just go to https://github.com/settings/keys to add your key? step 1 and 2 should be automatically covered by the notebook startup script.

Reply:
Oh, is adding a key to https://github.com/settings/keys the same as `ssh-add -K ~/.ssh/id_ed25519` ? If so, SG, thanks